docs/css/bulma-docs.css

```diff
@@ -6542,6 +6538,14 @@ a.panel-block:hover {
   border-bottom-color: #00d1b2;
 }
 
+.footer .sticky {
```

Comment:
Shouldn't this be `.footer.sticky` since you're expecting the `sticky` class to be on the same element that has the `footer` class?